conference/NW/nsdi.yml

```diff
@@ -4,14 +4,14 @@
   rank: A
   dblp: nsdi
   confs:
-    - year: 2022
-      id: nsdi22
-      link: https://www.usenix.org/conference/nsdi22/
+    - year: 2023
```

Comment:
Please keep the history instead of edit it, like this:
https://github.com/ccfddl/ccf-deadlines/blob/aae68886d4991d9648683603e0cf5e8ef9f02673/conference/DB/sigmod.yml#L6-L26

Reply:
History is added.